src/transformers/file_utils.py

```diff
@@ -81,6 +81,31 @@
     _torch_tpu_available = False
 
 
+try:
+    import psutil  # noqa: F401
```

Comment:
This is a fairly innocuous dependency afaict, and you could add it to requirements.txt/setup.py